flink-ml-benchmark/src/main/resources/benchmark-conf-kmeans.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
```

Comment:
Would it be better to rename these two files as `kmeans-benchmark.json` and `naivebayes-benchmark.json`,  so that the file names are a bit more concise, and it is a bit easier to spot the key differentiator (i.e. algorithm name) between these files?

How about we rename `benchmark-conf.json` as `benchmark-demo.json` to clarify the difference between this file and other files?